mitosheet/src/components/taskpanes/AITransformation/aiUtils.tsx

```diff
@@ -39,4 +40,25 @@ export const getChatHeight = (userInput: string, chatInputRef: React.MutableRefO
     } else {
         return chatOrScrollMax;
     }
-}
\ No newline at end of file
+}
+
+
+export const reconIsColumnCreated = (columnHeader: ColumnHeader, uiState: UIState, sheetData: SheetData): boolean => {
```

Comment:
Rather than these functions taking UIState, can they just take an optional recon - I think this makes more sense.